src/bin/tikv-server.rs

```diff
@@ -60,6 +62,22 @@ use tikv::raftstore::store::keys::region_raft_prefix_len;
 use tikv::pd::RpcClient;
 use tikv::util::time_monitor::TimeMonitor;
 
+const RAFTCF_MIN_MEM: u64 = 256 * 1024 * 1024;
+const RAFTCF_MAX_MEM: u64 = 2 * 1024 * 1024 * 1024;
+const KB: u64 = 1024;
+const DEFAULT_BLOCK_CACHE_RATIO: &'static [f64] = &[0.4, 0.15, 0.01];
```

Comment:
should use bit shift instead of float operation,
for alignment with MB / GB boundary.